server/cluster/cluster.go

```diff
@@ -352,6 +341,37 @@ func (c *RaftCluster) Start(s Server) error {
 	return nil
 }
 
+func (c *RaftCluster) runServiceCheckJob() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	var once sync.Once
+
+	checkFn := func() {
+		if c.isAPIServiceMode {
+			once.Do(c.initSchedulers)
+			c.enabledServices.Store(mcsutils.SchedulingServiceName, true)
```

Comment:
As described in this code, scheduling will not start again in api mode. I think we can delete runServiceCheckJob first and put it in the next PR. Because once the PR merged, the scheduling service must be deployed

Reply:
make sense